src/ontology/mondo.Makefile

```diff
@@ -1298,6 +1302,24 @@ update-subclass-sync: $(TMPDIR)/subclass-confirmed.robot.owl tmp/subclass-axioms
 		make NORM
 		mv NORM $(SRC)
 
+tmp/synonyms-axioms.owl: $(SRC)
+	$(ROBOT) filter --input $(SRC) --term oboInOwl:hasExactSynonym --axioms annotation --preserve-structure false --trim false \
+		--drop-axiom-annotations "oboInOwl:source=~'(DOID|ICD10CM|icd11.foundation|NCIT|OMIM|OMIMPS|Orphanet):.*'" \
+		-o $@
+
+# This command updates mondo-edit with all the confirmed synonyms evidence from the mondo-ingest repo
+.PHONY: update-synonyms-sync
+update-synonyms-sync: $(TMPDIR)/synonyms-confirmed.robot.owl tmp/synonyms-axioms.owl
+	grep -vE '^synonym:.*EXACT' $(SRC) > tmp/$(SRC)
```

Comment:
why only EXACT?

Reply:
My bad, good we have reviews.. Fixed now!